src/app/grievance-outbound-call-allocation/grievance-outbound-call-allocation.component.spec.ts

```
@@ -0,0 +1,25 @@
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ GrievanceOutboundCallAllocationComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(GrievanceOutboundCallAllocationComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)